### PR TITLE
Adds new icons for FoW

### LINKED ISSUE #1276: Add new icons needed for the FoW experiments and a status icon
### Area

UI Components

### The problem

There are several icons we need to add to the library. Some icons are needed to support the current selectables work in progress, others are required to support the 'physical' implementation of the design system as part of the FoW experiments.

### The solution

**Gender**
- Male
- Female
- Gender neutral
- Disabled access / wheelchair user

**Facilities**
- Restaurant/Food
- Snack (e.g. a sweet, a chocolate, or an apple maybe)
- Shop or store front
- Hospital (or more generally medical/medicine)
- First-aid kit

**Status**
- We need a solid variant of the success icon, specifically for use with the checkbox and switch components

### Alternatives and examples
- https://storybook.saltdesignsystem.com/?path=/docs/documentation-icons-icon--page
- https://github.com/jpmorganchase/salt-ds/tree/main/packages/icons/src/SVG

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

----------

```[tasklist]
### Tasks
- [ ] https://github.com/jpmorganchase/salt-ds/issues/1339
- [ ] https://github.com/jpmorganchase/salt-ds/issues/1347
```

### PR DESCRIPTION
**Pull request by origami-z**
TODO:
- [x] decide whether success solid is a breaking change or not 

Adds following icons
- Accessible
- Cart
- Crops
- Github
- Hospital
- Man
- Man woman
- Map
- Marker
- Medical kit
- Picnic
- Pot food
- Signpost
- Stethoscope
- Storefront
- Success
- Tote
- Universal access
- Utensils
- Woman
- Woodland

Closes #1276

### CODE REVIEW

**Comment by origami-z** [comment]
@pseys success-fill changed completely, which will be a breaking change...?

File: packages/icons/src/SVG/success-solid.svg
Comment (origami-z) [inline packages/icons/src/SVG/success-solid.svg:1]: This is a breaking change needs discussion

Comment (pseys) [reply packages/icons/src/SVG/success-solid.svg:1]: Is there any way for us to know if success-solid is being used anywhere currently? Its not included in any of our components.

**Comment by pseys** [comment]
@origami-z I'm not sure why but medical-kit-solid isn't displaying correctly in Storybook. It seems fine in Figma.

**Comment by origami-z** [comment]
> @origami-z I'm not sure why but medical-kit-solid isn't displaying correctly in Storybook. It seems fine in Figma.

Yes, i will re-check everything again.

File: packages/icons/src/SVG/success-solid.svg
Comment (origami-z) [reply packages/icons/src/SVG/success-solid.svg:1]: @joshwooding do we treat this as breaking change?

**Comment by origami-z** [comment]
Checked everything
- removed stroke color on medical-kit-solid in Figma
- remove clip content on component layer setting to avoid unnecessary `clipPath` introduced (also added to the salt icon contribution checker plugin)
@pseys

File: packages/icons/src/SVG/success-solid.svg
Comment (joshwooding) [reply packages/icons/src/SVG/success-solid.svg:1]: Hmm, I would say no but I think it's up to design whether it's a breaking change or not. The change is drastic but I can't remember how visual changes are being treated cc @bhoppers2008

**Review by alycrys — APPROVED** [review]

File: packages/icons/src/SVG/success-solid.svg
Comment (bhoppers2008) [reply packages/icons/src/SVG/success-solid.svg:1]: I think we can avoid the discussion around what constitutes a breaking change here. Given success-solid looks the same as success tick looks the same as success, let's class this is a design bug that we're rectifying and go ahead and merge.

**Pull request merged**